src/main/java/com/znsio/e2e/tools/Driver.java

```diff
@@ -340,4 +340,8 @@ public WebElement waitTillElementIsVisible (String locator) {
     public WebElement waitTillElementIsVisible (String locator, int numberOfSecondsToWait) {
         return (new WebDriverWait(driver, numberOfSecondsToWait)).until(ExpectedConditions.visibilityOf(findElementByAccessibilityId(locator)));
     }
+
+    public void setWindowSize (int width, int height) {
```

Comment:
where and how will this be used?

Reply:
there are use cases where a new tab gets opened. For such cases running on CI, we need to set window size. So I thought of creating a method for it.
If needed we can add a check of Driver.type before setting the window size

Reply:
The check will be good - else it may create problems

Reply:
added